Flashcards.Dejmenek/Flashcards.Dejmenek/Controllers/FlashcardsController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Flashcards.Dejmenek.DataAccess.Repositories;
+using Flashcards.Dejmenek.Helpers;
+using Flashcards.Dejmenek.Models;
+using Flashcards.Dejmenek.Services;
+
+namespace Flashcards.Dejmenek.Controllers
```

Comment:
Can you change to [file scoped namespace](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)? Please do this for all files that are blocked scoped like this one 😅